odoo/addons/base/tests/test_views.py

```diff
@@ -4589,8 +4590,14 @@ def test_uncommented_invisible_field(self):
             _logger.info('%s\n%s', msg, msg_info)
 
         if module_error_views:
-            msg_info = '\n'.join(f'Addons: {module!r}   Views: {names}' for module, names in module_error_views.items())
-            _logger.error('%s\n%s', msg, msg_info)
+            error_lines = []
+            for module, view_errors in module_error_views.items():
+                error_lines.extend([f"Addon: {module!r}"])
+                error_lines.extend([
+                    f"{' ' * 3}View: {view}\n{' ' * 6}Fields:\n" + "\n".join(f"{' ' * 9}{field}" for field in fields)
+                    for view, fields in view_errors.items()
+                ])
```

Comment:
I think it is easier to read like this:
```suggestion
                error_lines.append(f"Addon: {module!r}")
                for view, fields in view_errors.items():
                    error_lines.extend([f"{' ' * 3}View: {view}", f"{' ' * 6}Fields:"])
                    error_lines.extend(f"{' ' * 9}{field}" for field in fields)
```